### PR TITLE
Display metadata closes #49

### LINKED ISSUE #49: Show song information

### PR DESCRIPTION
**Pull request by Jaapp-**
Display metadata of the current song in rooms.
Comes with https://github.com/MoodCat/MoodCat.me-Core/pull/138

### CODE REVIEW

**Comment by JeremybellEU** [comment]
Shouldn't the .main.js.swp be omitted? it is a binary file.

File: app/scripts/controllers/main.js [hunk 1]
```diff
@@ -148,6 +148,36 @@ angular.module('moodCatApp')
         author: 'System'
       };
 
+      var song = $scope.room.song;
+      song.artworkUrl = song.artworkUrl.replace('-large', '-t500x500');
```
Comment (JeremybellEU) [inline app/scripts/controllers/main.js]: When this is null, an exception is thrown and the frontend crashes.

File: app/scripts/controllers/main.js [hunk 2]
```diff
@@ -167,11 +197,13 @@ angular.module('moodCatApp')
         //Clear the chat input field
         $scope.chatMessage.message = '';
 
+		$scope.hasPurchase = true;
```
Comment (JeremybellEU) [inline app/scripts/controllers/main.js]: This might not be true, is there some way we can determine it?

File: app/scripts/controllers/main.js [hunk 3]
```diff
@@ -148,6 +148,36 @@ angular.module('moodCatApp')
...
+
+	  $scope.makeTimeStamp = function(time) {
+		  time = Math.floor(time);
+		  var timeSeconds = time / 1000;
+		  var seconds = Math.floor(timeSeconds % 60);
+		  var minutes = Math.floor((timeSeconds % 3600) / 60);
+		  var hours = Math.floor(timeSeconds / 3600);
+          return pad(hours, 2) + ':' + pad(minutes, 2) + ':' + pad(seconds, 2);
+	  };
+
```
Comment (JeremybellEU) [inline app/scripts/controllers/main.js]: The above functions, already exist in MoodcatAudio.js. Can't we reuse them?

File: app/scripts/app.js [hunk 4]
```diff
@@ -49,12 +49,7 @@ angular
             rooms : ['$stateParams', 'roomService', 'soundCloudService', '$q', function($stateParams, roomService, soundCloudService, $q) {
               return roomService.fetchRooms($stateParams.mood).then(function(rooms) {
                 return $q.all(rooms.map(function(room) {
-                  return soundCloudService
-                    .fetchMetadata(room.song.soundCloudId)
-                    .then(function(data) {
-                      room.data = data;
-                      return room;
-                    });
+                  return room;
```
Comment (JWGmeligMeyling) [inline app/scripts/app.js]: There is no reason to map anymore. Just return `roomService.fetchRooms($stateParams.mood)`

**Comment by Azkle** [comment]
image

It's not yet scalable

**Comment by Azkle** [comment]
I can't seem to click on the purcase url, I seem to be under the footer.

**Comment by Azkle** [comment]
LGTM

**Pull request merged**